stripe.go

```diff
@@ -144,9 +143,9 @@ func SetBackend(backend SupportedBackend, b Backend) {
 }
 
 // Call is the Backend.Call implementation for invoking Stripe APIs.
-func (s BackendConfiguration) Call(method, path, key string, form *url.Values, params *Params, v interface{}) error {
+func (s BackendConfiguration) Call(method, path, key string, form Form, params *Params, v interface{}) error {
 	var body io.Reader
-	if form != nil && len(*form) > 0 {
```

Comment:
I removed the `len` check here because it's problematic for our interface above, and both `Encode` implementations will encode safely to an empty string which is not harmful in an HTTP request regardless of whether it's in a query string or body.